I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 22, in script
    jump start_game
ScriptError: could not find label 'start_game'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 22, in script
    jump start_game
  File "C:\Users\Acer\Desktop\renpy-8.3.2-sdk\renpy\ast.py", line 1699, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\Acer\Desktop\renpy-8.3.2-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start_game'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.2.24090902
visual novel 0.1
Fri Nov 22 22:51:41 2024
